c++/src/capnp/any.h

```diff
@@ -465,6 +467,12 @@ class AnyStruct::Builder {
 
   inline operator Reader() const { return Reader(_builder.asReader()); }
   inline Reader asReader() const { return Reader(_builder.asReader()); }
+
+  template <typename T>
+  BuilderFor<T> as() {
+    return typename T::Builder(_builder);
+  }
+  // T must be a struct type.
```

Comment:
Unwritten KJ style rule: This comment should be inside the function block, on the first line after the '{'.

Also on line 442.